update.mjs

```diff
@@ -94,6 +94,7 @@ const env = [
 ]
 
 const workspaces = [
+  '.gitmojirc.json',
```

Comment:
Can you add it to the end of the list? Thanks

Reply:
Done! 👍